test/listmodel.test.js

```diff
@@ -16,14 +16,33 @@ describe('list tests', function () {
     })
   })
   describe('active tests', function () {
-    it('should be able to get active value', function () {
+    xit('should be able to get active value', function () {
       const list1 = new List('list1')
       assert.strictEqual(list1.active, true)
     })
-    it('should be able to set active to false', function () {
+    xit('should be able to set active to false', function () {
       const list1 = new List('list1')
       list1.toggleActive()
```

Comment:
Bin nicht so ganz zufrieden mit der Benamung der Methode, aber ansonsten ist es OK.

Reply:
Vielleicht musst du mir kurz erklären, was die Methode toggleActive() machen soll. Auch in Bezug auf toggleDone().

Reply:
Ich finde irgendwas in die Richtung setInactive klarer von Benennung.

Reply:
`toggleDone()` soll einen Task auf erledigt oder unerledigt setzen je nach aktuellem Status. (Somewhere inside Task: done = false oder done = true)

`toggleActive()` soll eine Liste auf aktiv oder inaktiv je nach aktuellem Status.(Somewhere inside List: active = false oder active = true) Ich hatte zuerst `setActive(true/false)` überlegt, fand dann aber `toggleActive()` safer (ohne Argument).

Vielleicht sagt dir die css-Methode element.classList.toggle(class) was? Vermutlich hatte ich diese Benennung im Kopf. 

Hast du ein anderes Verständnis davon was mit active in List bzw. done in Task passieren soll?

Reply:
Ok, dann passt das für mich :)